ts/parts/Interaction.ts

```diff
@@ -291,10 +291,20 @@ TrackerMixin = H.TrackerMixin = {
             pointer = chart.pointer,
             renderer = chart.renderer,
             snap = (chart.options.tooltip as any).snap,
+            tooltip = chart.tooltip,
             tracker = series.tracker,
-            i,
-            onMouseOver = function (): void {
-                if (chart.hoverSeries !== series) {
+            i: number,
+            onMouseOver = function (e: Highcharts.PointerEventObject): void {
+
+                pointer.normalize(e);
+
+                if (
+                    chart.hoverSeries !== series &&
+                    (
+                        !tooltip ||
+                        !tooltip.isStickyOnContact()
```

Comment:
Is this related to #11635, or is it a random issue related to the `stickOnContact` feature? Ideally, this PR should be focused on only the changes required to fix #11635. We can leave it for now, I just want to understand the connection.

Reply:
This comes from an old merge during the first try of solving the issue.